kolibri/plugins/coach/assets/src/views/lessons/LessonSummaryPage/index.vue

```diff
@@ -94,7 +94,6 @@
   import CoachAppBarPage from '../../CoachAppBarPage';
   import ReportsControls from '../../common/ReportsControls';
   import { REPORTS_LESSON_TABS_ID, ReportsLessonTabs } from '../../../constants/tabsConstants';
-  import { PageNames } from '../../../constants';
```

Comment:
I think I'd rather maintain the explicit import here and reference the imported object.

I know this is technically available by virtue of the commonCoachMixin - but I'd prefer our direction of travel to be moving away from implicit uses of that, rather than towards.